### PR TITLE
[Documentation] Add docs for permissions and ENV vars

### PR DESCRIPTION
**Pull request by bjrne**
Are there any env variables missing? Maybe someone with more knowledge of priorities when choosing values from .env files and bash contexts can add a paragraph about that.

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -8,14 +8,25 @@ Environment variables
 -------------------------
 We use environment variables for:
 
-* Configuration
+* Configuration (mapview)
+* Credentials
 
-  * choosing a mapview backend that delivers the map tiles (square pngs of the map at a certain zoom level) to the user. For more technical information, refer to :ref:`architecture:mapview`.
+This is a complete list of environment variables we use. Their documentation is documented further in their respective topic:
```
Comment (bjrne) [inline docs/configuration.rst:14]: Is it though? Do we use more?

Comment (Baschdl) [reply docs/configuration.rst:14]: I cannot think of any other variables except the ones used in travis testing logic

**Review by Baschdl — APPROVED** [review]
Nicely written and it renders 👍

**Comment by Baschdl** [comment]
@bjrne Did you leave the same sections on purpose in the README?

**Comment by bjrne** [comment]
For now, yes, as we still work with it. A new readme is part of an extra issue, #33. And also some paragraphs still contain helpful information for writing other parts of the documentation, so I thought I'd leave it in. Replacing the whole Readme in the end is easy enough :)

**Pull request merged**